client/css/widgets/scoreboard.css

```diff
@@ -0,0 +1,44 @@
+.scoreboard {
+  box-sizing: border-box;
+  background: #ffffff;
+  color: #ffffff;
+  border: 2px solid black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.scoreboard table {
+  padding: none;
+  border:  none;
+  border-collapse: collapse;
+  display: block;
+  overflow-x: scroll;
+  overflow-y: scroll;
+}
+
+.scoreboard .firstRow {
+  position: sticky;
+  top: 0;
+}
+
+.scoreboard .firstCol {
+  position: sticky;
+  left: 0;
```

Comment:
You should also ensure that at least `.firstCol` and `.firstRow` have a background set (though you could give all cells a background) so that they occlude the text of the cells behind them.

Reply:
Fixed, thanks.